docs/api-diffs/9.0/rc.1/diff.md

```diff
@@ -0,0 +1,1290 @@
++            BlockingSequential = 0,
++            NonBlockingConcurrent = 3,
++            NonBlockingSequential = 2,
++        }
+-        [ExperimentalAttribute("ASPIREEVENTING001", UrlFormat="https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+-        public interface IDistributedApplicationEvent
++        public interface IDistributedApplicationEvent
+         public interface IDistributedApplicationEventing {
++            Task PublishAsync<T>(T @event, EventDispatchBehavior dispatchBehavior, CancellationToken cancellationToken = default(CancellationToken)) where T : IDistributedApplicationEvent;
+-            [ExperimentalAttribute("ASPIREEVENTING001", UrlFormat="https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+-            Task PublishAsync<T>(T @event, CancellationToken cancellationToken = default(CancellationToken)) where T : IDistributedApplicationEvent;
++            Task PublishAsync<T>(T @event, CancellationToken cancellationToken = default(CancellationToken)) where T : IDistributedApplicationEvent;
+-            [ExperimentalAttribute("ASPIREEVENTING001", UrlFormat="https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+-            DistributedApplicationEventSubscription Subscribe<T>(IResource resource, Func<T, CancellationToken, Task> callback) where T : IDistributedApplicationResourceEvent;
++            DistributedApplicationEventSubscription Subscribe<T>(IResource resource, Func<T, CancellationToken, Task> callback) where T : IDistributedApplicationResourceEvent;
+-            [ExperimentalAttribute("ASPIREEVENTING001", UrlFormat="https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+-            DistributedApplicationEventSubscription Subscribe<T>(Func<T, CancellationToken, Task> callback) where T : IDistributedApplicationEvent;
++            DistributedApplicationEventSubscription Subscribe<T>(Func<T, CancellationToken, Task> callback) where T : IDistributedApplicationEvent;
+-            [ExperimentalAttribute("ASPIREEVENTING001", UrlFormat="https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+-            void Unsubscribe(DistributedApplicationEventSubscription subscription);
++            void Unsubscribe(DistributedApplicationEventSubscription subscription);
+         }
+-        [ExperimentalAttribute("ASPIREEVENTING001", UrlFormat="https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+-        public interface IDistributedApplicationResourceEvent : IDistributedApplicationEvent
++        public interface IDistributedApplicationResourceEvent : IDistributedApplicationEvent
+     }
+ }
+```
+
+## Aspire.Hosting.Analyzers
+
+``` diff
+ {
++    namespace Aspire.Hosting.Analyzers {
++        [DiagnosticAnalyzerAttribute("C#", new string[]{ })]
++        public class AppHostAnalyzer : DiagnosticAnalyzer {
++            public AppHostAnalyzer();
++            public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; }
++            public override void Initialize(AnalysisContext context);
++        }
++    }
++}
```

Comment:
Can we filter this out? This is not real API - it is a tooling (analyzer) assembly.